PYME/recipes/processing.py

```diff
@@ -336,7 +335,6 @@ class WavefrontVelocity(ModuleBase):
         
     
     def execute(self, namespace):
-        from skimage.measure import profile_line
```

Comment:
Same as previous comment - used in code which is currently commented out for debugging, but not safe to remove.